src/Project/HabitatHome/website/Sitecore.HabitatHome.Commerce.Website.csproj

```diff
@@ -62,6 +62,9 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="Microsoft.CSharp" />
+    <Reference Include="Sitecore.Support.40718">
+      <HintPath>..\..\..\..\lib\Support\Sitecore.Support.40718.dll</HintPath>
```

Comment:
You reference the DLL - should that be committed?

Reply:
yep! one sec